bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/thing/check-control-channel-types.xml

```diff
@@ -6,13 +6,25 @@
 	<channel-type id="checkcontrol-name-channel">
 		<item-type>String</item-type>
 		<label>CheckControl Description</label>
+		<tags>
+			<tag>Control</tag>
+			<tag>Channel</tag>
```

Comment:
I don't know what to use here. This is a car service status message.

Reply:
`Status` or `Status.Info` ??